raiden/ui/app.py

```diff
@@ -188,6 +188,7 @@ def run_app(
         network_id,
         environment_type,
         unrecoverable_error_should_crash,
+        patfinding_service_address,
```

Comment:
here and below there is a typo `patfinding` -> `pathfinding`

Reply:
done